addons/stock/models/stock_move_line.py

```diff
@@ -9,6 +9,16 @@
 from odoo.tools.float_utils import float_compare, float_is_zero, float_round
 from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
 
+def get_aggregated_properties(move_line=False, move=False):
```

Comment:
Why put this method outside of the class ?
It isn't better to make it an `@api.model` method in case we could have to override it in the future ?